providers/base/units/stress/boot.pxu

```diff
@@ -229,21 +185,7 @@ template-unit: job
 plugin: shell
 environ: LD_LIBRARY_PATH
 command:
-  lspci -i "$SNAP"/usr/share/misc/pci.ids > "$PLAINBOX_SESSION_SHARE"/lspci_test
-  nmcli -t -f active,BSSID d w l | grep -oP "(?<=^yes:).*" > "$PLAINBOX_SESSION_SHARE"/wifi_test
-  checkbox-support-lsusb -f "$CHECKBOX_RUNTIME"/var/lib/usbutils/usb.ids -s | sort > "$PLAINBOX_SESSION_SHARE"/lsusb_test
-  if ! diff -u "$PLAINBOX_SESSION_SHARE"/lspci_original "$PLAINBOX_SESSION_SHARE"/lspci_test; then
-    echo "lspci mismatch during cycle {reboot_id}"
-    exit 1
-  fi
-  if ! diff -u "$PLAINBOX_SESSION_SHARE"/wifi_original "$PLAINBOX_SESSION_SHARE"/wifi_test; then
-    echo "wifi mismatch during cycle {reboot_id}"
-    exit 1
-  fi
-  if ! diff -u "$PLAINBOX_SESSION_SHARE"/lsusb_original "$PLAINBOX_SESSION_SHARE"/lsusb_test; then
-    echo "lsusb mismatch during cycle {reboot_id}"
-    exit 1
-  fi
+  reboot_check_test.sh -c "$PLAINBOX_SESSION_SHARE/before_reboot" -d "$PLAINBOX_SESSION_SHARE/warm_reboot_cycle{reboot_id}" -s
```

Comment:
Much nicer than the big piece of Shell script embedded in the job that was here before! :)